### ISSUE TITLE
Cannot find source file:      mtextedit.h

### ISSUE THREAD

**Issue by J-Dunn** [realthunder/FreeCAD#282]
```
-- Configuring done
CMake Error at src/Mod/TechDraw/Gui/CMakeLists.txt:387 (add_library):
  Cannot find source file:

    mtextedit.h

```
Looks similar to: https://github.com/FreeCAD/FreeCAD/commit/968faa511a  bit things have changed a little since that fix.

**Comment by realthunder** [comment]
It seem the referred commit has already been merged. Are you compiling LinkDaily or LinkStage3 branch?

**Comment by J-Dunn** [comment]
Thanks.  I initially did a cmake with an incorrect -S option ( it pointed to a FreeCAD/FreeCAD tree ). I then did make with the correctly source specified thunder tree and got this error.  Since this bug was removed years ago I don't quite see where this came from, but deleting the entire thunder tree and gitting a new clone cleared the problem.

Maybe there is cmake command I could have run clear the CMake files instead of just deleting. 
Anyway sorry for the noise, it does not seem to be bug in your work.
